### PR TITLE
fix: Fail when no id is sent while getting profiles

### PR DESCRIPTION
**Pull request by agusaldasoro**

### CODE REVIEW

File: src/LambdasClient.ts [hunk 1]
```diff
@@ -36,6 +36,9 @@ export class LambdasClient implements LambdasAPI {
   }
 
   fetchProfiles(ethAddresses: string[], profileOptions?: ProfileOptions, options?: RequestOptions): Promise<any[]> {
+    if (ethAddresses.length === 0) {
+      throw new Error('At least one id must be present when requesting profiles')
```
Comment (menduz) [inline src/LambdasClient.ts:40]: Return [] instead?

**Review by menduz — COMMENTED** [review]
Functions are maps between two domains, there should be no reason to fail mapping an empty array to an empty array. It may be the responsibility of the controller or handler to fail instead

File: src/LambdasClient.ts (hunk 1 above)
Comment (agusaldasoro) [reply src/LambdasClient.ts:40]: yes, better!

Comment (agusaldasoro) [reply src/LambdasClient.ts:40]: Should we refactor all the lambdas client then? https://github.com/decentraland/catalyst-client/blob/22248f778bf7983bfcd00882881d2fadc1e37e6c/src/LambdasClient.ts#L68

🤔

Comment (menduz) [reply src/LambdasClient.ts:40]: I'd say if applied consistently, that approach is safer. Maybe it doesn't need to change everywhere at once tho

**Review by marianogoldman — APPROVED** [review]

**Pull request merged**